src/core/qgsvectorlayer.cpp

```diff
@@ -3867,6 +3867,16 @@ QStringList QgsVectorLayer::uniqueStringsMatching( int index, const QString &sub
 }
 
 QVariant QgsVectorLayer::minimumValue( int index ) const
+{
+  return minOrMax( index, true );
+}
+
+QVariant QgsVectorLayer::maximumValue( int index ) const
+{
+  return minOrMax( index, false );
+}
+
+QVariant QgsVectorLayer::minOrMax( int index, bool minimum ) const
```

Comment:
maybe rename this function to be a bit more explicit (even though it's not part of the public API)? minimumOrMaximumValue(...) ?